keras/callbacks.py

```diff
@@ -428,6 +429,48 @@ def on_epoch_end(self, epoch, logs=None):
                     self.model.save(filepath, overwrite=True)
 
 
+class DeadReluDetector(Callback):
+    def __init__(self, original_model, x_train):
+        if not isinstance(original_model, Sequential):
```

Comment:
@gewoonrik 
Afterwards I am calling `model.layers` and this is defined in Sequential, not in base class. Although I can imagine other models with layers, eg something with skip-connections like ResNet, for now AFAIK Sequential is the only *real* model for Keras